vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -106,6 +106,27 @@ controller, and serviceaccounts controller.`,
 				os.Exit(1)
 			}
 		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			verflag.PrintAndExitIfRequested()
+			utilflag.PrintFlags(cmd.Flags())
+
+			c, err := s.Config(KnownControllers(), ControllersDisabledByDefault.List())
+			if err != nil {
+				return err
```

Comment:
Can't we keep the exits?  Is it always a problem or just when there's a bug in teh binary?

Reply:
although that's equal with `glog.Fatal` so maybe we should be good with it